src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
```

Comment:
```suggestion
    private Long id;
```
why should we use wrapper?

Reply:
Oops
If necessary, we can use methods, as well as use in collections

Reply:
the default value of long is 0 - and we won't understand if it is missing or has value 0